default/stringtables/en.txt

```diff
@@ -1394,6 +1394,19 @@ Sets the path for the directory containing the AI script files, for current exec
 OPTIONS_DB_AI_CONFIG
 Is available to the AI via the freeorioninterface, is set for current execution only.  Current expected use is to name an optional AI config file within the AI script folder; default is the empty string.  Intended to facilitate AI testing.
 
+OPTIONS_DB_AI_CONFIG_BEHAVIOR_AGGRESSION_FORCED
```

Comment:
This changes are not related to:  Change stringtable suffixes associated with aggression to be consistent